charts/wildfly/values.schema.json

```diff
@@ -594,13 +594,499 @@
+                      "format": "int32",
+                      "type": [
+                        "integer",
+                        "null"
+                      ]
+                    },
+                    "periodSeconds": {
+                      "description": "How often (in seconds) to perform the probe. Default to 10 seconds. Minimum value is 1.",
+                      "format": "int32",
+                      "type": [
+                        "integer",
+                        "null"
+                      ]
+                    },
+                    "successThreshold": {
+                      "description": "Minimum consecutive successes for the probe to be considered successful after having failed. Defaults to 1. Must be 1 for liveness and startup. Minimum value is 1.",
+                      "format": "int32",
+                      "type": [
+                        "integer",
+                        "null"
+                      ]
+                    },
+                    "tcpSocket": {
+                      "required": [
+                        "port"
+                      ],
+                      "type": "object",
+                      "description": "TCPSocket specifies an action involving a TCP port.",
+                      "properties": {
+                        "host": {
+                          "description": "Optional: Host name to connect to, defaults to the pod IP.",
+                          "type": "string"
+                        },
+                        "port": {
+                          "description": "Number or name of the port to access on the container. Number must be in the range 1 to 65535. Name must be an IANA_SVC_NAME.",
+                          "oneOf": [
+                            {
+                              "type": "string"
+                            },
+                            {
+                              "type": "integer"
+                            }
+                          ]
+                        }
+                      }
+                    },
+                    "terminationGracePeriodSeconds": {
+                      "description": "Optional duration in seconds the pod needs to terminate gracefully upon probe failure. The grace period is the duration in seconds after the processes running in the pod are sent a termination signal and the time when the processes are forcibly halted with a kill signal. Set this value longer than the expected cleanup time for your process. If this value is nil, the pod's terminationGracePeriodSeconds will be used. Otherwise, this value overrides the value provided by the pod spec. Value must be non-negative integer. The value zero indicates stop immediately via the kill signal (no opportunity to shut down). This is a beta field and requires enabling ProbeTerminationGracePeriod feature gate. Minimum value is 1. spec.terminationGracePeriodSeconds is used if unset.",
```

Comment:
\> The value zero indicates stop immediately via the kill signal
\> Minimum value is 1
stopping immediately is not an option ?

Consider 
"If this value is nil OR UNSET, the pod's terminationGracePeriodSeconds will be used"
and removing "spec.terminationGracePeriodSeconds is used if unset."," to make desc shorter

Reply:
good idea